src/pages/gateway/headers.md

```diff
@@ -87,7 +93,15 @@ This can be useful for authorization, authentication, and tracking headers that
 -  **Key**: `GGW-SH-differentSource-trackingId`
    -  **Value**: `different-trackingId`
 
-## Add response headers
+## Response headers
+
+Response headers provide detailed context of the response. Currently, you can only add response headers from the mesh config.
+
+<InlineAlert variant="info" slots="text"/>
+
+All header names must be unique. Conflicting header names will result in an error.
```

Comment:
```suggestion
All header names must be unique. Conflicting header names will result in an error on mesh creation and update.
```

Also FYI, this won't be true until we implement [CEXT-30](https://jira.corp.adobe.com/browse/CEXT-30) (and only for response headers configured at the source level)